### PR TITLE
CI: remove Julia-*.app before running tests

### PR DESCRIPTION
**Pull request by bevanjkay**
Remove Julia-*.app before running tests.

Currently blocking https://github.com/Homebrew/homebrew-cask-versions/pull/12688

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -62,6 +62,10 @@ jobs:
               echo '::warning::Removing Visual Studio is no longer necessary.'
             fi
           fi
+          
+          if ! rm -r /Applications/Julia-*.app; then
+            echo '::warning::Removing Julia is no longer necessary.'
+          fi
```
Comment (carlocab) [inline .github/workflows/ci.yml:68]: I think we can actually do `brew uninstall --cask julia` here.

**Review by carlocab — DISMISSED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bevanjkay) [reply .github/workflows/ci.yml:68]: @carlocab Is it definitely the `julia` cask, just not on the latest version? And would it still be formatted as follows or without the if statement?

```
          if ! brew uninstall --cask julia; then
            echo '::warning::Removing Julia is no longer necessary.'
          fi
```

Comment (carlocab) [reply .github/workflows/ci.yml:68]: Yes, I'm pretty sure it's the casks. I have some workflows in my tap that do `brew uninstall --cask julia` too.

We can try something like
```
          if ! brew uninstall --cask julia && ! rm -r /Applications/Julia-*.app; then
            echo '::warning::Removing Julia is no longer necessary.'
          fi
```

Comment (bevanjkay) [reply .github/workflows/ci.yml]: 👍🏻 Done

**Review by gdams — APPROVED** [review]

**Pull request merged**